<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inserttable_class</class>
 <widget class="QDialog" name="inserttable_class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>297</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>265</width>
    <height>297</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="rows">
         <property name="text">
          <string>Rows: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="rows_spinbox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="columns">
         <property name="text">
          <string>Columns:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="columns_spinbox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="spacing">
         <property name="text">
          <string>Cell spacing: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spacing_spinbox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="padding">
         <property name="text">
          <string>Cell padding:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="padding_spinbox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Insert</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
